extensions/cql/standard/clause_2_conformance.adoc

```diff
@@ -58,7 +58,7 @@ The specific set of operators defined in this conformance class are:
 * Spatial operators:
 ** intersects
 * Temporal operators:
-** anyinteracts
+** <<req_simple-cql_temporal-predicate,anyinteracts>>
```

Comment:
I don't think we should add a link for one of the predicates. Either we do it for all of them or for none of them.

Reply:
Since we don't link to the definition for any other operator, I removed the reference.  Instead I added a note indicating that _anyinteracts_ is an operator defined in this specification in requirement /req/simple-cql/temporal-predicate.